common/Linux/ScreenSaverDBus.cpp

```diff
@@ -0,0 +1,109 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2010  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <dbus/dbus.h>
+#define BUS_NAME "org.freedesktop.ScreenSaver"
+#define BUS_PATH "/org/freedesktop/ScreenSaver"
+#define BUS_INTERFACE "org.freedesktop.ScreenSaver"
+static DBusConnection* connection = nullptr;
+static dbus_uint32_t cookie;
+bool ChangeScreenSaverStateDBus(bool inhibit_requested = true, char* program_name = (char*)"Unknown", char* reason = (char*)"Unknown")
```

Comment:
Use `const char*` if you don't need to modify

Also, putting the default values here doesn't do anything.  Put them in the header file if you want the default values.